spec/features/cash_register_proficiency_checking_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Cash register proficiency checking :', type: :feature do
+  let(:member) { create :member }
+
+  before { sign_in member }
+
+  describe 'cash_register_proficiency_checking' do
+    context 'when the cash register proficiency is insufficient and it one place are still available on time slot' do
```

Comment:
`'when the cash register proficiency is insufficient, and there is only one slot left on a given time_slot'`